doc/interfaces/serial.rst

```diff
@@ -2,9 +2,18 @@
 
 CAN over Serial
 ===============
-
 A text based interface. For example use over bluetooth with
-``/dev/rfcomm0``
+``/dev/rfcomm0`` or with Arduino ``/dev/ttyACM0``. The interface is a simple
+implementation that is primarily intended for recording CAN traces.
```

Comment:
Instead of saying the interface has a "primary intended use" say it "has been used for".